start_crashtest_codeship.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
is there no `.yml` file necessary for codeship?

Reply:
Unfortunately, for Codeship Basic, there is no YAML file possible to be configured. It needs to be done via their Frontend.